lab-michelle/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-michelle",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "cowsay": "^1.2.1",
+    "nodemon": "^1.11.0",
```

Comment:
Small details, but don't forget to validate that your dependencies are in the right category. Superagent and Nodemon are both devDeps.